ipam/allocator.go

```diff
@@ -73,6 +74,7 @@ type Allocator struct {
 	isKnownPeer       func(mesh.PeerName) bool
 	quorum            func() uint
 	now               func() time.Time
+	monitor           monitor.Monitor // monitor tracks changes in address ranges owned by us
```

Comment:
The fact that you need a comment here is a good indication that either the type name or the var name or both are not as descriptive as they should be.